modules/patching-ovnk-address-ranges.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="patching-ovnk-address-ranges_{context}"]
+= Patching OVN-Kubernetes address ranges
+
+OVN-Kubernetes reserves the following IP address ranges:
+
+* `100.64.0.0/16`. This IP address range is used for the `internalJoinSubnet` parameter of OVN-Kubernetes by default. 
+* `100.88.0.0/16`. This IP address range is used for the `internalTransSwitchSubnet` parameter of OVN-Kubernetes by default.
+
+If these IP addresses have been used by OpenShift SDN or any external networks that might communicate with this cluster, you must patch them to use a different IP address range before initiating the limited live migration.
+
+The following procedure can be used to patch CIDR ranges that are in use by OpenShift SDN if the migration was initially blocked.
+
+[NOTE]
+====
+This is an optional procedure and must only be used if the migration was blocked after using the `oc patch Network.config.openshift.io cluster --type='merge' --patch '{"metadata":{"annotations":{"network.openshift.io/network-type-migration":""}},"spec":{"networkType":"OVNKubernetes"}}'` command "Initiating the limited live migration process".
```

Comment:
Is there maybe a missing "from" before the section title reference? Also suggest moving the long command to its own block.
```suggestion
This is an optional procedure and must only be used if the migration was blocked after using the following command from "Initiating the limited live migration process":

[source,terminal]
----
$ oc patch Network.config.openshift.io cluster --type='merge' \
    --patch '{"metadata":{"annotations":{"network.openshift.io/network-type-migration":""}},"spec":{"networkType":"OVNKubernetes"}}'` 
----
```